Gradata/AGENTS.md

```
@@ -0,0 +1,194 @@
+
+# MCP server (stdio, IDE integration)
+python -m gradata.mcp_server
+```
+
+## Architecture
+
+### Strict layering rule
+
+Lower layers **never** import from higher layers. Violations are bugs and should be flagged in code review.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)